src/test/test_unite_mocks.h

```diff
@@ -21,6 +24,19 @@
 
 namespace mocks {
 
+template <typename T>
+struct stub;
```

Comment:
This and the next seem not used.

Reply:
This is required as this is the general template which is specialized down there. Without it it there's nothing to specialize and compilation will fail.

The next one is indeed unused. The two specializations are for `const` and non-const functions. In case anyone wants to use this for other mocks and mock a non-const function and walk into a failure this would be quite hard to debug I guess, that's why I kept it (for completeness reasons).

I do buy into YAGNI (until you need it) though. So if you insist, I will remove.

Reply:
I would rather not have dead code :)

Reply:
Removed in https://github.com/dtr-org/unit-e/pull/1078/commits/49336d14ced1fae6325ece4cccab6fd681298a84